plugins/chat/spec/plugin_spec.rb

```diff
@@ -114,6 +115,34 @@
           expect(serializer.can_chat_user).to eq(false)
         end
       end
+
+      context "when both users are in Chat.allowed_group_ids" do
+        before do
+          SiteSetting.chat_allowed_groups = group.id
+          SiteSetting.direct_message_enabled_groups = group.id
+          GroupUser.create(user: target_user, group: group)
+          GroupUser.create(user: user, group: group)
+        end
+
+        it "returns true" do
```

Comment:
maybe it teturns true for valid users

Reply:
Yes that's correct, it returns true when users are valid.. I can update the test description to make it more precise